clang/lib/Sema/SemaBounds.cpp

```diff
@@ -4433,10 +4542,16 @@ namespace {
       return false;
     }
 
-    // If E is a (possibly parenthesized) lvalue variable V,
+    // If E is a possibly parenthesized lvalue variable V,
     // GetLValueVariable returns V. Otherwise, it returns nullptr.
+    //
+    // V may have value-preserving operations applied to it.  For example,
```

Comment:
It would be helpful in the comment to describe what value-preserving casts can be applied to lvalue expressions.